src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.hpp

```diff
@@ -48,6 +48,8 @@ T getValueFromTosaConst(mlir::Value &val) {
   return val.getDefiningOp<mlir::tosa::ConstOp>().getValue().cast<T>();
 }
 
+mlir::ElementsAttr getElementsAttrFromConst(mlir::Value &val);
```

Comment:
NIT: Maybe you can add a comment explaining that both onnx and tosa const ops are supported